packages/sdk/public/util.js

```
@@ -1,7 +1,7 @@
 /**
  * `Network` is a class that manages the network status.
  */
-export class Network {
+class Network {
```

ESLint

[error] 'Network' is defined but never used.

(@typescript-eslint/no-unused-vars)